Settings-test.toml

```diff
@@ -17,3 +17,11 @@ raster_data_root_path = "../test_data/raster" # relative to sub crate directory
 
 [upload]
 path = "test_upload"
+
+[oidc]
+enabled = true
+issuer = ""
+client_id = ""
+client_secret = ""
+redirect_uri = ""
+scopes = [ ]
```

Comment:
You should add the [oidc] entry to Settings-default.toml as well. You could set enabled to false, for example.

Reply:
f14d56f9228aaf2b12541b54b20e58253afbe282 0863ba86d02427fe0dabadd64755ae22d3cb9307